src/assets/scripts/util/autofit_images.js

```diff
@@ -1,241 +1,240 @@
-(function(){
-
-
-function getClosestWidth (element, baseNode) {
-	var elementWidth = element.offsetWidth;
-	var baseWidthCheck = 50;
-	var widthMaximum = 4800;
-	var diffFromWidth = widthMaximum;
-	var chosenNode = "";
-	for (var i = baseWidthCheck; i < widthMaximum; i+=50) {
-		if (Math.abs(elementWidth - i) < diffFromWidth && element && element.attributes && element.attributes[baseNode+i] && element.attributes[baseNode+i].nodeValue && element.attributes[baseNode+i].nodeValue.length > 1) {
-			diffFromWidth = elementWidth - i;
-			chosenNode = baseNode+i;
-		}
-	}
-	return chosenNode;
-}
-
-function getPosition(element) {
-    var yPosition = 0;
-
-    while(element) {
-        yPosition += (element.offsetTop - element.scrollTop + element.clientTop);
-        element = element.offsetParent;
-    }
+function initAutofitImages() {

```

Comment:
Is this the right one? I sent you a structure to use. because this isn't all about auto fitting images this is doing the randomizing and we're going to pull that out into its own module.